libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -103,6 +103,10 @@
         PUSH_AUTHENTICATION_FAILED,
         PUSH_AUTHENTICATION_IGNORED,
         SETTINGS_LANGUAGE_SELECTION_FORCED,
+        NOTIFICATION_SETTINGS_LIST_OPENED,
+        NOTIFICATION_SETTINGS_STREAMS_OPENED,
```

Comment:
Oh, I should have clarified. I'm going to use these in https://github.com/wordpress-mobile/WordPress-Android/issues/2525